uint8 command_id
uint8 status_code